package.json

```diff
@@ -17,16 +17,17 @@
     "postcss-normalize": "9.0.0",
     "postcss-preset-env": "6.7.0",
     "postcss-safe-parser": "5.0.2",
-    "react": "17.0.1",
+    "react": "^16.14.0",
```

Comment:
Did we drop down just for enzyme? I know enzyme for 17 is really close, but perhaps instead we should look at using the official testing library? https://github.com/testing-library/react-testing-library